templates/bridgeserver2.yaml

```diff
@@ -137,6 +152,57 @@ Parameters:
 Conditions:
   CreateDevResources: !Equals [ !Ref BridgeEnv, dev ]
 Resources:
+  DBCluster:
+    Type: AWS::RDS::DBCluster
+    Properties:
+      AvailabilityZones: !Ref DatabaseClusterAvailabilityZones
+      BackupRetentionPeriod: !Ref DatabaseBackupRetentionDays
+      DatabaseName: 'BridgeDB'
+      DBClusterIdentifier: !Join
+        - '-'
+        - - !Ref 'AWS::StackName'
+          - aurora
+      DBSubnetGroupName: !ImportValue us-east-1-bridgeserver2-common-DBSubnetGroupName
+      DeletionProtection: true
+      Engine: 'aurora'
+      EngineMode: 'provisioned'
+      EngineVersion: '5.6.10a'
+      MasterUsername: !Ref HibernateConnectionUsername
+      MasterUserPassword: !Ref HibernateConnectionPassword
+      Port: 3306
+      PreferredMaintenanceWindow: Mon:10:00-Mon:10:30
+      StorageEncrypted: true
+      VpcSecurityGroupIds:
+        - !ImportValue us-east-1-BridgeServer2-vpc-VpnSecurityGroup
+        - !Ref RdsNewVpcSecurityGroup
+  DBInstance:
+    Type: AWS::RDS::DBInstance
+    Properties:
+      AllowMajorVersionUpgrade: false
+      AutoMinorVersionUpgrade: true
+      AvailabilityZone: !Ref DatabaseAvailabilityZone
+      DBClusterIdentifier: !Ref DBCluster
+      DBInstanceClass: !Ref DatabaseInstanceType
+      DBInstanceIdentifier: !Join
```

Comment:
Typically, allowing CFN to manage resource names makes resources more maintainable. I would recommend that unless there's a requirement to give it a specific name.